### PR TITLE
NFDIV-3284: Add common header and footer and remove family court logo

### PR DESCRIPTION
**Pull request by FaisalMoJ**
### Change description ###

Add common header and footer and remove family court logo

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/NFDIV-3284

**Before merging a pull request make sure that:**

- [X] tests have been updated / new tests has been added (if needed)
- [ ] README and other documentation has been updated / added (if needed)

**If this ticket will have any visible impact on users and is not behind a feature toggle, make sure that:**
- [ ] this ticket been reviewed by QA
- [ ] the user story been signed off by the PO

Note that bug fixes, dependency updates and technical tasks do not directly impact the user experience and can be merged without QA and PO review.

### CODE REVIEW

**Review by amoriartyMOJ — APPROVED** [review]

File: src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java [hunk 1]
```diff
@@ -73,7 +88,12 @@ void shouldReturnTemplateContentForSole() {
             entry(DATE_OF_HEARING, "8 November 2021"),
             entry(TIME_OF_HEARING, "14:56 pm"),
             entry(BEFORE_DATE_OF_HEARING, "1 November 2021"),
-            entry(HAS_FINANCIAL_ORDERS, true)
+            entry(HAS_FINANCIAL_ORDERS, true),
+            entry(CTSC_CONTACT_DETAILS, buildCtscContactDetails()),
+            entry(DIVORCE_AND_DISSOLUTION_HEADER, "Divorce and Dissolution"),
```
Comment (BWeeks-JusticeGovUK) [inline src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java:93]: use DIVORCE_AND_DISSOLUTION_HEADER_TEXT

File: src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java [hunk 2]
```diff
@@ -73,7 +88,12 @@ void shouldReturnTemplateContentForSole() {
...
+            entry(COURTS_AND_TRIBUNALS_SERVICE_HEADER, "HM Courts & Tribunals Service"),
```
Comment (BWeeks-JusticeGovUK) [inline src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java:94]: use COURTS_AND_TRIBUNALS_SERVICE_HEADER_TEXT

File: src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java [hunk 3]
```diff
@@ -73,7 +88,12 @@ void shouldReturnTemplateContentForSole() {
...
+            entry(CONTACT_EMAIL, "contactdivorce@justice.gov.uk"),
```
Comment (BWeeks-JusticeGovUK) [inline src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java:95]: use CONTACT_DIVORCE_EMAIL

File: src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java [hunk 4]
```diff
@@ -73,7 +88,12 @@ void shouldReturnTemplateContentForSole() {
...
+            entry(PHONE_AND_OPENING_TIMES, "Phone: 0300 303 0642 (Monday to Friday, 8am to 6pm)")
```
Comment (BWeeks-JusticeGovUK) [inline src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java:96]: use PHONE_AND_OPENING_TIMES_TEXT

File: src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java [hunk 5]
```diff
@@ -98,7 +122,12 @@ void shouldReturnTemplateContentForJoint() {
             entry(DATE_OF_HEARING, "8 November 2021"),
             entry(TIME_OF_HEARING, "14:56 pm"),
             entry(BEFORE_DATE_OF_HEARING, "1 November 2021"),
-            entry(HAS_FINANCIAL_ORDERS, true)
+            entry(HAS_FINANCIAL_ORDERS, true),
+            entry(CTSC_CONTACT_DETAILS, buildCtscContactDetails()),
+            entry(DIVORCE_AND_DISSOLUTION_HEADER, "Divorce and Dissolution"),
```
Comment (BWeeks-JusticeGovUK) [inline src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java:127]: See related comment

File: src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java [hunk 6]
```diff
@@ -98,7 +122,12 @@ void shouldReturnTemplateContentForJoint() {
...
+            entry(COURTS_AND_TRIBUNALS_SERVICE_HEADER, "HM Courts & Tribunals Service"),
```
Comment (BWeeks-JusticeGovUK) [inline src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java:128]: See related comment

File: src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java [hunk 7]
```diff
@@ -98,7 +122,12 @@ void shouldReturnTemplateContentForJoint() {
...
+            entry(CONTACT_EMAIL, "contactdivorce@justice.gov.uk"),
```
Comment (BWeeks-JusticeGovUK) [inline src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java:129]: See related comment

File: src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java [hunk 8]
```diff
@@ -98,7 +122,12 @@ void shouldReturnTemplateContentForJoint() {
...
+            entry(PHONE_AND_OPENING_TIMES, "Phone: 0300 303 0642 (Monday to Friday, 8am to 6pm)")
```
Comment (BWeeks-JusticeGovUK) [inline src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java:130]: See related comment

File: src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java [hunk 9]
```diff
@@ -238,4 +267,17 @@ private ConditionalOrderCourtDetails setupConditionalOrderCourtDetailsConfig() {
         when(conditionalOrderCourtDetailsConfig.get(BURY_ST_EDMUNDS.getCourtId())).thenReturn(expectedDetails);
         return expectedDetails;
     }
+
+    private CtscContactDetails buildCtscContactDetails() {
```
Comment (BWeeks-JusticeGovUK) [inline src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java:271]: We should add these values to DocmosisTemplateConstants so they can be utilised properly.

File: src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java (hunk 1 above)
Comment (FaisalMoJ) [reply src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java:93]: Removed magic strings and replaced with constants

File: src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java (hunk 3 above)
Comment (FaisalMoJ) [reply src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java:95]: Removed magic strings and replaced with constants

File: src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java (hunk 4 above)
Comment (FaisalMoJ) [reply src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java:96]: Removed magic strings and replaced with constants

File: src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java (hunk 5 above)
Comment (FaisalMoJ) [reply src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java:127]: Removed magic strings and replaced with constants

File: src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java (hunk 7 above)
Comment (FaisalMoJ) [reply src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java:129]: Removed magic strings and replaced with constants

File: src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java (hunk 6 above)
Comment (FaisalMoJ) [reply src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java:128]: Removed magic strings and replaced with constants

File: src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java (hunk 8 above)
Comment (FaisalMoJ) [reply src/test/java/uk/gov/hmcts/divorce/document/content/CertificateOfEntitlementContentTest.java:130]: Removed magic strings and replaced with constants

**Review by BWeeks-JusticeGovUK — APPROVED** [review]

**Pull request merged**